src/main/java/com/odysseusinc/arachne/portal/scheduler/TempHolder.java

```diff
@@ -47,10 +47,10 @@
     @Scheduled(cron = "${tmp.holder.cron}")
     public void hold() throws IOException {
         final File tmp = getTmpFolder();
-        if (tmp.exists() && tmp.isDirectory()) {
+        if (tmp.isDirectory()) {
             for (final File file : getTomcatFolders()) {
                 final File folder = new File(file , INNER_FOLDER);
-                if (folder.getParentFile().exists() && folder.isDirectory()) {
+                if (folder.isDirectory()) {
```

Comment:
@return <code>true</code> if and only if the file denoted by this
  abstract pathname exists <em>and</em> is a directory;
  <code>false</code> otherwise